### PR TITLE
Game slider

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

**Comment by vitsalis** [comment]
Depends #312

**Comment by vitsalis** [comment]
Video: https://www.youtube.com/watch?v=ncb-KDoLBjY&feature=youtu.be

**Comment by dionyziz** [comment]
Nice.

**Comment by dionyziz** [comment]
You're adding jquery-ui files to `static/images`; I don't think you want that.

File: static/script/game/view.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ var GameView = {
     },
     getMap: function() {
         var href = this.href;
+        var slider = this.slider;
```
Comment (dionyziz) [inline static/script/game/view.js]: Where is `this.slider` used?

File: static/script/game/view.js [hunk 2]
```diff
@@ -103,6 +104,10 @@ var GameView = {
 
             $( '.roundid' ).text( 'Round ' + roundid );
 
+            if ( !slider ) {
```
Comment (dionyziz) [inline static/script/game/view.js]: What is this check?

File: static/script/game/view.js [hunk 3]
```diff
@@ -177,6 +182,20 @@ var GameView = {
...
         $( '.previous a' ).click( GameView.getMap );
+        $( '.slider' ).slider( {
+            min: 0,
+            max: GameView.roundCount - 1,
+            value: $( '.roundid' ).attr( 'data-id' ),
+            stop: function( e, ui ) {
+                var gameid = $( '.gamemeta h2' ).attr( 'data-id' );
+                var href = GameView.makeUrl( gameid, ui.value );
+
+                GameView.getMap.call( {
```
Comment (dionyziz) [inline static/script/game/view.js]: Suggest making a new `loadMap` function; reserve `getMap` as the dispatcher. Then you don't need to do `this` acrobatics.

**Pull request merged**